packages/suite/src/components/wallet/CoinmarketFooter/index.tsx

```diff
@@ -116,7 +116,9 @@ const CoinmarketFooter = () => {
                                 </Link>
                             </BoxLeft>
                             <BoxRight>
-                                <Button variant="tertiary">invity.io</Button>
+                                <Link href="https://invity.io/" target="_blank">
```

Comment:
i think this url should be added here:
https://github.com/trezor/trezor-suite/blob/develop/packages/suite-desktop/src-electron/config.ts#L6
otherwise it will be blocked by desktop app

and maybe you could use `ExternalLink` component instead to be easier to track it in the future?

Reply:
We don't want to have the icons from ExternalLink component.

![image](https://user-images.githubusercontent.com/3112191/97241186-9b4c7080-17f0-11eb-9ce2-823cba3b0ff7.png)